_posts/2021-05-16-overview-of-architecture-review-processes.md

```diff
@@ -0,0 +1,253 @@
+  1. The ARB chair is the architecture review process advocate. The ARB chair is responsible for ensuring the effectiveness of the review.
+
+### Process
+
+The overall review process bases on a few general principles such as that a clearly defined problem statement as the basis of the architecture, reviews are open processes, and companies conduct reviews for the project’s benefit (see also: {% cite Maranzano2005 %} for the full list of five principles).
+
+The review process follows roughly four phases.
+The specific implementation may vary.
+  1. Screening Phase: The project team requests a review. The project staff and ARB review the request to determine whether a review would benefit the project. If a review should be conducted, the ARB selects a review angel (see also: previous section) to oversee the project’s review process.
+  1. Preparation Phase: The ARB selects a review team and a review leader. Togehter with the project team, they determine the number of reviews and the initial review’s date and agenda. The review team, project team, review leader, and review angel verify that the project that should be reviewed has a clearly defined problem statement driving the architecture and an appropriate documentation as a basis for the review.
+  1. Review Meeting Phase: During the review, the members of the project present the mentioned problem statement and how the proposed architecture solves it. The reviewers ask further questions in order to figure out issues they believe could make the project an incomplete or inadequate solution to the problem. Finally, the review team meets privately and generates a report. The report is presented to the project team. Furthermore, management alerts can be raised, etc.
+  1. Follow-Up Phase: The review team delivers the report with the review’s findings to the project team within certain time frame such as 15 days after the review meetings are done. The project team (and, if a management alert has been raised, the management) must respond within two weeks.
+
+For more details, we refer to {% cite Maranzano2005 %}.
+
+### Artifacts
+
+As artifacts, Maranzano et al. mention three different types of artifacts {% cite Maranzano2005 %}:
+  1. There are checklists for architects to prepare for the review and checklists for the reviewers to do the review and figure out more of the architecture. The checklists serve as a collection of organizational knowledge and should be maintained.
+  1. There is input to the review. This can be documentation of the system requirements, functional requirements, or informal documentation. Maranzano et al. mention specifically that the architecture should be designed based on a clear problem statement tackling the functional and qualitative requirements, the costs, and the timeline. A document describing these points may be a specific design document or something like that.
```

Comment:
```suggestion
  1. There is input to the review. This can be documentation of the system requirements, functional requirements, or informal documentation. The architecture should be designed based on a clear problem statement tackling the functional and qualitative requirements, the costs, and the timeline.
```

> A document describing these points may be a specific design document or something like that.

This is the first time in the classical review that you mention design document.
At this point, I don't know what this is.
For me, it reads that the requested artifact has no formatting rules. Hence, I would drop it.